_includes/heading-text.html

```diff
@@ -0,0 +1 @@
+<a class="heading-anchor" id="{{ include.text | slugify }}"></a>{{ include.text }} <a href="#{{ include.text | slugify }}">#</a>
```

Comment:
I think jekyll automagically makes the id the text if writing in markdown. We direct link without this step currently :guitar: